MVP/Resources/Translations.it.resx

```diff
@@ -432,4 +426,136 @@
+    <value>Numero di Downloads</value>
+  </data>
+  <data name="field_number_of_events" xml:space="preserve">
+    <value>Numero di Eventi</value>
+  </data>
+  <data name="field_number_of_events_participated" xml:space="preserve">
+    <value>Numero di Eventi ai quali hai partecipato</value>
+  </data>
+  <data name="field_number_of_feedbacks_provided" xml:space="preserve">
+    <value>Numero di Feedbacks forniti</value>
+  </data>
+  <data name="field_number_of_followers" xml:space="preserve">
+    <value>Numero di Followers</value>
+  </data>
+  <data name="field_number_of_mentees" xml:space="preserve">
+    <value>Numero di Allievi</value>
+  </data>
+  <data name="field_number_of_mentorship_activity" xml:space="preserve">
+    <value>Numero di Attività di Menthoring</value>
+  </data>
+  <data name="field_number_of_posts" xml:space="preserve">
+    <value>Numero di Posts</value>
+  </data>
+  <data name="field_number_of_projects" xml:space="preserve">
+    <value>Numero di Progetti</value>
+  </data>
+  <data name="field_number_of_samples" xml:space="preserve">
+    <value>Numero di Esempi</value>
+  </data>
+  <data name="field_number_of_sites" xml:space="preserve">
+    <value>Numero di Siti</value>
+  </data>
+  <data name="field_number_of_subscribers" xml:space="preserve">
+    <value>Numero di Iscritti</value>
+  </data>
+  <data name="field_number_of_talks" xml:space="preserve">
+    <value>Numero di Colloqui</value>
+  </data>
+  <data name="field_number_of_threads_moderated" xml:space="preserve">
+    <value>Numero di Threads moderati</value>
+  </data>
+  <data name="field_number_of_videos" xml:space="preserve">
+    <value>Numero di Video</value>
+  </data>
+  <data name="field_number_of_views" xml:space="preserve">
+    <value>Numero di Visualizzazioni</value>
+  </data>
+  <data name="field_number_of_visitors" xml:space="preserve">
+    <value>Numero di Visitatori</value>
+  </data>
+  <data name="field_projects" xml:space="preserve">
+    <value>Progetto(i)</value>
+  </data>
+  <data name="field_prs_issues_submissions" xml:space="preserve">
+    <value>Pull Requests/Issues/Submissions</value>
+  </data>
+  <data name="field_views_of_answers" xml:space="preserve">
+    <value>Visualizzazioni di risposte</value>
+  </data>
+  <data name="onboarding_1_description" xml:space="preserve">
+    <value>Gestisci tutte le attività della tua community dal palmo della tua mano.</value>
+  </data>
+  <data name="onboarding_1_title" xml:space="preserve">
+    <value>Semplice ed efficace</value>
+  </data>
+  <data name="onboarding_3_description" xml:space="preserve">
+    <value>Crea direttamente contributi basati su un URL negli appunti. Pre-compilare tutto ciò che possiamo in modo da poter aggiungere rapidamente nuovi contributi.</value>
+  </data>
+  <data name="onboarding_3_title" xml:space="preserve">
+    <value>Aggiunta veloce</value>
+  </data>
+  <data name="sign_in" xml:space="preserve">
+    <value>Sign in</value>
```

Comment:
This is actually sign in in Italian? :o

Reply:
We use some English technical words... like Default or Login. Is it possible to use Accedi. Let me know

Reply:
I'm fine either way, as long as it's clear to the average Italian :) I just thought maybe it was a forgotten text.

Reply:
No. Some technical words are better in English than Italian